File: manager/src/main/java/io/hydrosphere/serving/clouddriver/ServiceInstance.java
package io.hydrosphere.serving.clouddriver;

import lombok.Data;

/**
 *
 */
@Data
public class ServiceInstance {

    private String id;

    private String host;

    private RuntimeInstanceStatus status;

    private String statusText;

    public enum RuntimeInstanceStatus{
        UP,
        DOWN
    }
}

File: manager/src/main/scala/io/hydrosphere/serving/manager/model/ServiceInstance.scala
package io.hydrosphere.serving.manager.model

import io.hydrosphere.serving.manager.model.ServiceInstanceStatus.ServiceInstanceStatus

case class ServiceInstance(
  instanceId: String,
  host: String,
  appPort: Int,
  sidecarPort: Int,
  sidecarAdminPort: Int,
  serviceId: Long,
  status: ServiceInstanceStatus,
  statusText: Option[String]
)